invenio_previewer/templates/invenio_previewer/macros.html

```diff
@@ -54,7 +54,7 @@
         <td class="nowrap">{{ file.size }}</td>
         <td class="nowrap">
           <span class="pull-right">
-            <a class="btn btn-xs btn-default" href="{{ file.uri }}">
+            <a class="btn btn-xs btn-default" href="{{ url_for('invenio_records_ui.recid_download', pid_value=file.pid.pid_value, filename=file.file.key) }}">
```

Comment:
Thanks a lot @lnielsen ;)

However, the CSV previewer has the same issue. I will comment it in another message to show the line.